plasTeX/Renderers/PageTemplate/simpletal/simpleTAL.py

```diff
@@ -841,10 +841,11 @@ def addTag (self, tag, tagProperties={}):
 					'singletonTag'    - A boolean to indicate that this is a singleton flag
 		"""
 		# Add the tag to the tagStack (list of tuples (tag, properties, useMacroLocation))
+                tagProperties = tagProperties or {}
```

Comment:
I think these changes look fine.  I don't see why it would break anything and it's better to just get this out of the way now to prevent any future issues.